### PR TITLE
Databases week2

### PR DESCRIPTION
**Pull request by Hailegebriel22**

### CODE REVIEW

File: databases/week2/tasks_queries.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  WHERE id = 36;
+  
+  -- Delete a task
+  DELETE FROM task WHERE id=36;
+  
+  -- Part 3: More queries
+  -- email ends in @spotify.com
+  
+  SELECT * FROM `user`
+ WHERE `email` LIKE '%@spotify.com';
```
Comment (Elia-Git) [inline databases/week2/tasks_queries.sql]: Here I don't see which tasks were assigned to users of their email ends with ,@spotify.com... as the target is to find which tasks  are assigned to the user...
--Get all the tasks assigned to users whose email ends in @spotify.com
-- Tasks of users with email ends in @spotify.com
```
SELECT user.id,user.name,user.email, task.title
FROM task
JOIN user_task ON task.id = user_task.task_id
JOIN user ON user.id = user_task.user_id
WHERE user.email LIKE '%@spotify.com';
```

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Good job here, small fix!

**Review by Elia-Git — APPROVED** [review]
You are a pro! Keep the good spirit and try to organize your task a bit as I can see week1 and week2 were pushed into this PR.

File: databases/week2/Librarydb.sql [hunk 2]
```diff
@@ -0,0 +1,55 @@
+-- Create a Library database.
+
+CREATE DATABASE IF NOT EXISTS `Library`;
+  
+  CREATE TABLE    
+ Library.author (
```
Comment (Hanouj) [inline databases/week2/Librarydb.sql:6]: Hi Haile as for table names isn't better for Space to be replaced with Underscore —

**Review by Hanouj — COMMENTED** [review]
you did a great work in this assignment Haile, keep it up

File: databases/week2/Librarydb.sql (hunk 2 above)
Comment (Hailegebriel22) [reply databases/week2/Librarydb.sql:6]: where? there is no space it is the database name Library and table name author( Library.author)

Comment (Hailegebriel22) [reply databases/week2/Librarydb.sql:6]: where? there is no space it is the database name Library and table name author( Library.author)

Comment (Hailegebriel22) [reply databases/week2/Librarydb.sql:6]: where? there is no space it is the database name Library and table name author( Library.author)

**Pull request merged**